docs/reference/db-config.rst

```diff
@@ -4,5 +4,6 @@
 db/config.cc properties
 =======================
```

Comment:
@annastuchlik As we discussed during our call, we could update the heading and incorporating an adding introduction that clarifies where and how these properties can be edited (as part of this pull request, or as a separate issue).

Reply:
- I think the title should be changed to "Configuration Parameters".

- A short introduction we should add now (it can be extended in a follow-up PR):

  This section contains a list of properties that can be configured in _scylla.yaml_ - the main configuration file for ScyllaDB.

Reply:
@nyh We'll merge it as is. If you think the description should be extended, I'll open a follow-up PR, because everything else looks good now.